publish-extensions.js

```diff
@@ -134,9 +137,42 @@ const flags = [
         continue;
       }
 
+      if (msVersion) {
+        if (!extension.repository) {
+          if (extension.download) ()
```

Comment:
We care about all extensions not about only with download property.

Reply:
Sorry for that one, this one was left there for some reason, but a mistake for sure.